components/crud-web-apps/jupyter/backend/apps/common/form.py

```diff
@@ -135,6 +135,25 @@ def set_notebook_image_pull_policy(notebook, body, defaults):
     )
 
 
+def set_notebook_http_rewrite_uri(notebook, body, defaults):
```

Comment:
Lets remove this helper function since, for now, the backend should just add any necessary annotation based on the server type found in the request